test/e2e/framework/cmd_exec_operations.go

```diff
@@ -76,7 +76,7 @@ func (co *cmdOps) Exec(command string, opts ...E2EOption) (stdOut, stdErr *bytes
 	cmd.Stderr = &stderr
 	err = cmd.Run()
 	if err != nil {
-		return &stdout, &stderr, fmt.Errorf("error while running '%s', stdOut: %s, stdErr: %s, err: %s", command, stdout.String(), stderr.String(), err.Error())
+		return &stdout, &stderr, fmt.Errorf("error while running '%s' (detailed arguments: %q), stdOut: %s, stdErr: %s, err: %s", command, cmdArgs, stdout.String(), stderr.String(), err.Error())
```

Comment:
@chandrareddyp This will change the output for any error in the E2E tests.
Are you ok with that?

Reply:
The `command` includes arguments, so there is no need to add `cmdArgs` in the logs as it is redundant.

Reply:
Here is an error message where listing the arguments show an error:

```
"error while running 'tanzu context create --endpoint http://localhost:8080 --staging plugin-sync-tmc-ke9C  --force-csp=true' (detailed arguments: [\"context\" \"create\" \"--endpoint\" \"http://localhost:8080\" \"--staging\" \"plugin-sync-tmc-ke9C\" \"\" \"--force-csp=true\"]), stdOut: , stdErr: [x] : accepts at most 1 arg(s), received 2\n, err: exit status 1"
```
Notice the extra space before `--force-csp` in the command printout.  When trying this on the shell locally, it is not a problem.  But if you look at the list of arguments, which is what the tests really use, there is an extra _**empty**_ argument before `--force-csp=true`, and that makes the code fail.

That is why I suggest printing the list of arguments.

Reply:
Okay got it.

Reply:
Thanks.  I reverted the breaking change that illustrated the problem and will merge once CI passes.